### ISSUE TITLE
Task: List of upcoming events

### ISSUE THREAD

**Issue by flaki** [rust-community/events-team#12]
We need to figure out:
- What kind of events we need to track (is it just conferences, or meetups as well, only meetup groups etc)
- What should be the visibility/accessibility of the end result (we may need to track events that haven't been announced yet, so a public list might be problematic in this case)
- How to solicit data? Who will add new events? (the event hosts themselves, or is this something curated by the events team.)
- What form will the data be in? (shared calendar? form+spreadsheet? etherpad? GitHub issue/issue template?)

**Comment by skade** [comment]
This list now exists, but is private due to personal data included. Please get in touch for access.
